src/main/java/servlets/module/challenge/XxeChallenge1.java

```diff
@@ -62,7 +62,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // Setting IpAddress To Log and taking header for original IP if forwarded from proxy
     ShepherdLogManager.setRequestIp(request.getRemoteAddr(), request.getHeader("X-Forwarded-For"));
     log.debug(LEVEL_NAME + " Servlet Accessed");
-    response.setContentType("text/html");
```

Comment:
Should we not set the content type?

Reply:
You are right, actually there is no reason to remove this.